### DISCUSSION TITLE
Problem: HTTP Request von “Open DTU on Battery” an “Shelly 3EM Pro” liefert  Fehler 404.

### DISCUSSION THREAD

**Discussion by expansiv** [hoylabs/OpenDTU-OnBattery#561] [Q&A] [answered]
Guten Tag,

Problem: HTTP Request /status  von “Open DTU on Battery” an “Shelly 3EM Pro” liefert  Fehler 404.
Anbei Shelly 3EM Pro Log mit Fehler: shelly_http_server.:263 No handler for '/status‘
Frage:
Ist die Shelly 3EM Pro überhaupt für die Kommunikation mit “Open DTU on Battery” zugelassen / berücksichtigt? Anscheinend kennt die Shelly Pro z. B.: den http Befehl „status“ nicht?
Oder
Wie müsste der http  Befehl lauten?

Herzlichen Dank

[2023_12_26 Konsole.pdf](https://github.com/helgeerbe/OpenDTU-OnBattery/files/13773776/2023_12_26.Konsole.pdf)

**Comment by stefan123t** [comment]
@expansiv der endpoint bei Shelly 3EM vs. 3EM Pro hat sich geändert.

`http://<shelly3em-123456789abc>/status` -> `http://<shellypro3em-123456789abc>/rpc/Shelly.GetStatus`

Siehe auch die Issues #246, #251, #364 und #438 zur `DIGEST` Authentication und zum Parsen der JSON Antwort.

Speziell laut https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/247#discussioncomment-6352035 sollte sowohl DIGEST Auth als auch ein JSON Path, z.B. `/em:0/total_act_power` funktionieren.

**Reply by expansiv** [reply]
Success! Vielen Dank. Ich informiere den DTU Lieferanten. Er muss seine Info für die Einstellungen anpassen.

**Reply by schlimmchen** [reply, marked as answer]
@expansiv Bitte markiere die entsprechende Antwort als Lösung für diese Diskussion.

**Reply by greymda** [reply]
@expansiv  can you post the updated file for 3EM Pro, please?
